docs/development/frontend.rst

```diff
@@ -92,3 +92,5 @@ specific. This tightly controls the cascade of styles.
 The majority of the SCSS styles are found within the 'blocks' layer,
 with each BEM block in its own file. All blocks are documented at the top of
 the file to provide guidelines for use and modification.
+
+One of these blocks provides code syntax highlighting, which can be tested on reference project provided at `<http://localhost/project/pypi-code-highlighting-demo/>`_ when using dev database dump. Source rst file is available `here <https://github.com/evemorgen/pypi-code-highlighting-demo>`_.
```

Comment:
Can you wrap this line at 80 characters and change:
  * "tested on reference project" to "tested with a reference project"
  * "dev database dump" to "development database"
  * "rst" to "reStructuredText"?

Reply:
Sure thing, I think I addressed all the issues in #104a452